src/kOS/Screen/KOSToolBarWindow.cs

```diff
@@ -80,100 +80,65 @@ public class KOSToolBarWindow : MonoBehaviour
+            
         }
 
         /// <summary>
         /// Unity hates it when a MonoBehaviour has a constructor,
         /// so all the construction work is here instead:
         /// </summary>
-        public void FirstTimeSetup()
+        public static void FirstTimeSetup()
         {
-            ++countInstances;
-            myInstanceNum = countInstances;
-            SafeHouse.Logger.SuperVerbose("KOSToolBarWindow: Now making instance number "+myInstanceNum+" of KOSToolBarWindow");
-
-            const string LAUNCHER_BUTTON_PNG = "GameData/kOS/GFX/launcher-button.png";
-            const string TERMINAL_OPEN_ICON_PNG = "GameData/kOS/GFX/terminal-icon-open.png";
-            const string TERMINAL_CLOSED_ICON_PNG = "GameData/kOS/GFX/terminal-icon-closed.png";
-            const string TERMINAL_OPEN_TELNET_ICON_PNG = "GameData/kOS/GFX/terminal-icon-open-telnet.png";
-            const string TERMINAL_CLOSED_TELNET_ICON_PNG = "GameData/kOS/GFX/terminal-icon-closed-telnet.png";
-
-            // ReSharper disable SuggestUseVarKeywordEvident
-            WWW launcherButtonImage = new WWW("file://" + KSPUtil.ApplicationRootPath.Replace("\\", "/") + LAUNCHER_BUTTON_PNG);
-            WWW terminalOpenIconImage = new WWW("file://" + KSPUtil.ApplicationRootPath.Replace("\\", "/") + TERMINAL_OPEN_ICON_PNG);
-            WWW terminalClosedIconImage = new WWW("file://" + KSPUtil.ApplicationRootPath.Replace("\\", "/") + TERMINAL_CLOSED_ICON_PNG);
-            WWW terminalOpenTelnetIconImage = new WWW("file://" + KSPUtil.ApplicationRootPath.Replace("\\", "/") + TERMINAL_OPEN_TELNET_ICON_PNG);
-            WWW terminalClosedTelnetIconImage = new WWW("file://" + KSPUtil.ApplicationRootPath.Replace("\\", "/") + TERMINAL_CLOSED_TELNET_ICON_PNG);
-            // ReSharper enable SuggestUseVarKeywordEvident
-            launcherButtonImage.LoadImageIntoTexture(launcherButtonTexture);
-            terminalOpenIconImage.LoadImageIntoTexture(terminalOpenIconTexture);
-            terminalClosedIconImage.LoadImageIntoTexture(terminalClosedIconTexture);
-            terminalOpenTelnetIconImage.LoadImageIntoTexture(terminalOpenTelnetIconTexture);
-            terminalClosedTelnetIconImage.LoadImageIntoTexture(terminalClosedTelnetIconTexture);
-
-            windowRect = new Rect(0,0,width,height); // this origin point will move when opened/closed.
+            launcherButtonTexture = GameDatabase.Instance.GetTexture ("kOS/GFX/launcher-button",false);
+            terminalOpenIconTexture = GameDatabase.Instance.GetTexture ("kOS/GFX/terminal-icon-open", false);
+            terminalClosedIconTexture = GameDatabase.Instance.GetTexture ("kOS/GFX/terminal-icon-closed", false);
+            terminalOpenTelnetIconTexture = GameDatabase.Instance.GetTexture ("kOS/GFX/terminal-icon-open-telnet", false);
+            terminalClosedTelnetIconTexture = GameDatabase.Instance.GetTexture ("kOS/GFX/terminal-icon-closed-telnet", false);
+                
```

Comment:
At first I didn't like this change away from using constant strings for the names, but now that I think of it, there's nowhere else we'd be using those strings anyway except for right here, so I guess this is okay.